provider/pkg/provider/util.go

```diff
@@ -115,7 +115,11 @@ func getActiveClusterFromConfig(config *clientapi.Config, overrides resource.Pro
 		currentContext = val.StringValue()
 	}
 
-	activeClusterName := config.Contexts[currentContext].Cluster
+	activeContext := config.Contexts[currentContext]
+	if activeContext == nil {
+		return &clientapi.Cluster{}
```

Comment:
This means we don't report up a useful error about what caused this. Should this function return a useful error?

Reply:
I'm not sure it's possible to tell if this is actually an error since this runs during preview and Outputs may not yet be fully resolved. I suspect that it might lead to erroneous warnings.

This particular case was a bit odd because the outdented YAML still parsed cleanly, but changed the semantics of the configuration. This change fixes the panic, and now correctly shows a replacement in the preview. This should be sufficient for a user to notice that something is going wrong if they weren't expecting a change.